source/includes/fundamentals/code-snippets/agg-factory.java

```diff
@@ -0,0 +1,17 @@
+Aggregates.match(
```

Comment:
Not a big deal, but the filename standard I was using was to use ".rst" for code snippets (e.g. auth-credentials-default.rst") and ".java" (using camelcase to adhere to Java class naming standards, but is not visible in the srv.java file) for full classes.

Reply:
Quite right